flexget/plugins/urlrewrite_divxatope.py

```diff
@@ -8,24 +8,59 @@
 from flexget.utils import requests
 from flexget.utils.soup import get_soup
 
+from flexget.entry import Entry
+from flexget.utils.search import torrent_availability, normalize_unicode
+
 log = logging.getLogger('divxatope')
 
+CATEGORIES = {
```

Comment:
With this part of the code:
```
'properties': {
            'category': {'type': 'string'},
},
```
you need to set this in the yml. Is there a way to avoid adding any property?
```
divxatope: 
  - category: all
```
I want to simplify to "divxatope", without properties.